src/Resources/config/services.yml

```diff
@@ -8,6 +8,11 @@ services:
     arguments:
       - '@request_stack'
       - '@inspiredminds.fieldsetduplication.helper.field'
+    tags:
+      - { name: contao.hook, hook: loadFormField, method: onLoadFormField }
+      - { name: contao.hook, hook: compileFormFields, method: onCompileFormFields, priority: -100}
+      - { name: contao.hook, hook: storeFormData, method: onStoreFormData, priority: 100 }
```

Comment:
Hm, why  a priority of 100 in this case in particular?

Reply:
I wanted to give it a high priority, because it was set to load before other hooks with the `array_unshift` method in `config.php`.

Reply:
Ah yes, good point 👍